pom.xml

```diff
@@ -99,6 +99,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
+            <scope>provided</scope>
```

Comment:
Maybe you can reorganize related to scopes, I can see ws-commons is in runtime section but compile scope...

Reply:
okay